xml/System.Workflow.Activities.Configuration/ActiveDirectoryRoleFactoryConfiguration.xml

```diff
@@ -102,7 +102,7 @@
       </AssemblyInfo>
       <Attributes>
         <Attribute>
-          <AttributeName>System.Configuration.ConfigurationProperty("DistiguishedName", DefaultValue=Mono.Cecil.CustomAttributeArgument)</AttributeName>
+          <AttributeName>System.Configuration.ConfigurationProperty("DistinguishedName", DefaultValue=Mono.Cecil.CustomAttributeArgument)</AttributeName>
```

Comment:
This is the literal string in source code, so it should be left as is.

Reply:
OK, it's been undone